docs/python-getting-started/index.md

```diff
@@ -0,0 +1,190 @@
+   tests were skipped completely.
+
+   <img src="unittest04.png">
+
+9. Feel free to experiment with other functions in the source tree and see what
+   happens, then go ahead and use `yourbase` in your own project!
+
+## Accelerate Tests in CI
+
+To use YourBase in your CI, you will need to set up a few things first in order
+to enable shared Dependency Graph storage. Once you have done that, you can add
+`yourbase` to your project (via `requirements.txt` or whatever mechanism you use
+to install your dependencies) and then use YourBase to accelerate the builds in
+your CI system.
+
+### About the Shared Dependency Graph
+
+To get acceleration gains across team members, you can store the Dependency
+Graph in the cloud and share it with others working on the same code-base.
+YourBase will use information from your project's commit history to determine
+the optimal Dependency Graph for each build. As a result, when you've submitted
+code that is the same as code already tested by your colleagues, YourBase will
+be able to skip those tests anywhere that has access to the shared graph
+storage.
+
+Currently, YourBase supports [S3 buckets][] for graph storage, so you will
+need to create a new (or have an existing) bucket available for storing your
+team's graphs. The graphs are separated by unique project, so you can choose to
+use one bucket per project or one bucket for all of your projects. In order to
+access the shared storage, anywhere that you will use YourBase needs to have
+credentials that have read, write and list permissions to the relevant S3
+bucket.
+
+[S3 buckets]: https://docs.aws.amazon.com/AmazonS3/latest/userguide/UsingBucket.html
+
+### S3 Configuration for CI
+
+YourBase needs the information required to access your S3 bucket; as mentioned
+before you will need to configure credentials (via environment, EC2 role,
+configuration file, etc.) in a way that the default configuration can access
+them. In addition to that you will need to export the S3 bucket information;
+below is an example of how to do this — more information can be found on the
+[PyPI documentation page][].
+
+1. Get the name of the bucket you want to use, for example
+   `acmecorp-yourbase-graphs`
+2. Export the `YOURBASE_REMOTE_CACHE` variable to be `s3://your-bucket-name`.
+   For example: `export YOURBASE_REMOTE_CACHE="s3://acmecorp-yourbase-graphs"`
+3. Configure your AWS credentials - you can do this one of two ways:
+    1. The standard AWS way (config file, or environment variable, etc), or
+    2. Using a YourBase-specific pair of environment variables (in case you want
+       to have a separate set of credentials just for the graph access), which
+       are `YOURBASE_AWS_ACCESS_KEY_ID` and `YOURBASE_AWS_SECRET_ACCESS_KEY`
+
+## Support for Parallelized Tests
+
+If you've already conducted your CI to parallelize tests, YourBase has you
+covered. YourBase can be configured to work with tests run in cohorts.
```

Comment:
Guess we never filled this in!

```suggestion
## Support for Parallelized Tests

If you've already conducted your CI to parallelize tests, YourBase has you
covered. YourBase can be configured to work with tests run in cohorts.

1. Set `YOURBASE_COHORT_COUNT` to the number of shards (or processes, or swarms, etc.) that run your tests. For example:
    ```sh
    export YOURBASE_COHORT_COUNT=12
    ```
2. Set `YOURBASE_ACTIVE_COHORT` to the ID of the current shard, in the range `[1, $YOURBASE_COHORT_COUNT]`. For example:
    ```
    # First shard
    export YOURBASE_ACTIVE_COHORT=1
    # ...
    # Last shard
    export YOURBASE_ACTIVE_COHORT=12
    ```